src/app/components/ms-identifier/ms-identifier.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="msIdentifier">
+    <p class="projectInfo-sectionSubHeader">{{ 'msidentifier' | translate }}</p>
+    <div>
+        <evt-project-info-section *ngIf="data?.settlement.length>0 || data?.country.length>0 || data?.region.length>0" [label]="'place'" [sectionClass]="'settlement'">
+            <span content>   
+                <evt-content-viewer *ngFor="let s of data.settlement" [content]="s"></evt-content-viewer>
```

Comment:
can data be undefined since you  checked with `data?.xxx` before?
The same comment applies for every `data.xxx` occurrence.